### DISCUSSION TITLE
Regarding the addition of support for personal local NAS storage functionality (such as Synology and QNAP).

### DISCUSSION THREAD

**Discussion by tasty007** [spacedriveapp/spacedrive#2676] [General]
### Check for other issues

- [X] I've checked for other issues, and none exist.

### Issue description

Have you considered supporting personal local NAS functionality, such as browsing personal local storage data on Synology and QNAP devices?

### Platform and versions

- OS:
- Spacedrive:

<details><summary> Development tool versions (optional) </summary><pre>

</pre></details>

### Code of Conduct

- [X] I agree to follow the Code of Conduct.

**Comment by linear** [comment]
<p><a href="https://linear.app/spacedriveapp/issue/ENG-1858/regarding-the-addition-of-support-for-personal-local-nas-storage">ENG-1858 Regarding the addition of support for personal local NAS storage functionality (such as Synology and QNAP).</a></p>

**Comment by iLynxcat** [comment]
Migrated this to a discussion; we do have alpha-stage support for some NAS devices and some of us internally are using it on TrueNAS but I'm not sure how well everything works on other NAS systems like Synology. I'll ask one of our team members to reply here soon!

**Comment by Rsfinder** [comment]
i'm using DS1821+ (synology) and am able to import locations into Spacedrive. 
does this mean that other models may have errors?

but i'm also currently experiencing that network folders are prone to disconnection when there are many folders/files.
